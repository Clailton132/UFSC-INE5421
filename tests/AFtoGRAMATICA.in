a,b,c
q0;q0;q1;qr
q1;q1;q2;qr
q2;q0;q0;q2
qr;qr;qr;qr
f:q1
i:q0
