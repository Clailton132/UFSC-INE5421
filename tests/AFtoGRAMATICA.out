q0:aq0|b|bq1|;q1:a|aq1|bq2|;q2:aq0|bq0|cq2;
